pom.xml

```diff
@@ -50,4 +50,13 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>maven-plugin</artifactId>
+      <version>[2.0,)</version>
```

Comment:
It would be great to make this dependency optional. Many people do not like the plugin & would use the opportunity to unbundle it in Jenkins 2.0